src/api/procedures/allowIdentityToCreatePortfolios.ts

```diff
@@ -0,0 +1,66 @@
+import { PolymeshError, Procedure } from '~/internal';
+import { ErrorCode } from '~/types';
+import { ExtrinsicParams, TransactionSpec } from '~/types/internal';
+import { stringToIdentityId } from '~/utils/conversion';
+
+/**
+ * @hidden
+ */
+export type AllowIdentityToCreatePortfoliosParams = {
+  did: string;
+};
+
+/**
+ * @hidden
+ */
+export async function prepareAllowIdentityToCreatePortfolios(
+  this: Procedure<AllowIdentityToCreatePortfoliosParams, void>,
+  args: AllowIdentityToCreatePortfoliosParams
+): Promise<TransactionSpec<void, ExtrinsicParams<'portfolio', 'allowIdentityToCreatePortfolios'>>> {
+  const {
+    context: {
+      polymeshApi: {
+        tx: {
+          portfolio: { allowIdentityToCreatePortfolios },
+        },
+        query,
+      },
+    },
+    context,
+  } = this;
+  const { did } = args;
```

Comment:
We should asset the Identity/did exists. Usually the chain will accept it, but if the Identity isn't already made then its almost certainly a mistake.